diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..4db48e0
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,34 @@
+# Setting files
+.idea
+.vscode
+
+# OS X
+.DS_Store
+
+# build files
+build
+
+# Compiled Object files
+*.o
+*.obj
+*.slo
+*.lo
+
+# Compiled Dynamic libraries
+*.so
+*.dylib
+*.dll
+
+# Compiled Static libraries
+*.a
+*.la
+*.lai
+*.lib
+
+# Executables
+*.exe
+*.out
+*.app
+
+# DB files
+*.db
\ No newline at end of file
diff --git "a/doc/Lab1\345\256\236\351\252\214\346\226\207\346\241\243.md" "b/doc/Lab1\345\256\236\351\252\214\346\226\207\346\241\243.md"
index 56b1a7e..1b367c6 100644
--- "a/doc/Lab1\345\256\236\351\252\214\346\226\207\346\241\243.md"
+++ "b/doc/Lab1\345\256\236\351\252\214\346\226\207\346\241\243.md"
@@ -14,7 +14,7 @@
 
 1. 从record模块入手，先完成记录的序列化和反序列化
 2. 对于page模块，设计记录页面的整体组织形式
-3. 在table模块中，结合record模块和page模块，完成各项有关记录操作的借口 
+3. 在table模块中，结合record模块和page模块，完成各项有关记录操作的接口
 
 ## Lab1 需要实现的接口
 
@@ -42,7 +42,7 @@
     - SearchRecord 记录条件检索
     - NextNotFull 更新非满页面的页面编号，必要时插入新的页面
 
-所有需要实现的部分在上述各个函数的 **//Lab1 Begin** 到 **// Lab 1 End** 之间，建议按照模块说明中的上面所列的文件顺序依次实现。其中各个模块中具有 **//TIPS:** 标记来提示大家一些实现思路，以及需要重点关注的 **//ALERT:** 标记表示非常重要的提示。
+所有需要实现的部分在上述各个函数的 **// Lab1 Begin** 到 **// Lab 1 End** 之间，建议按照模块说明中的上面所列的文件顺序依次实现。其中各个模块中具有 **// TIPS:** 标记来提示大家一些实现思路，以及需要重点关注的 **// ALERT:** 标记表示非常重要的提示。
 
 ## Lab1 报告说明
 
diff --git "a/doc/\345\256\236\351\252\214\346\241\206\346\236\266.md" "b/doc/\345\256\236\351\252\214\346\241\206\346\236\266.md"
index 913cf7e..fd8665a 100644
--- "a/doc/\345\256\236\351\252\214\346\241\206\346\236\266.md"
+++ "b/doc/\345\256\236\351\252\214\346\241\206\346\236\266.md"
@@ -2,6 +2,7 @@
 
 ## 模块划分
 实验框架中各个模块的源代码整理到了各自独立的文件夹内。
+
 0.  minios
 
 - 功能：模拟底层OS系统，为数据库提供虚拟的持久化存储功能。
diff --git "a/doc/\351\207\215\350\246\201\345\243\260\346\230\216.md" "b/doc/\351\207\215\350\246\201\345\243\260\346\230\216.md"
index 03ff153..5917e26 100644
--- "a/doc/\351\207\215\350\246\201\345\243\260\346\230\216.md"
+++ "b/doc/\351\207\215\350\246\201\345\243\260\346\230\216.md"
@@ -23,6 +23,8 @@ git fetch upstream
 git merge upstream/lab2
 ```
 
+merge 后可能会产生冲突（如果你和上游仓库对同样的地方进行了不同的修改），这种情况下请手动解决所有冲突并执行 git 的 add 和 commit 指令完成合并。
+
 2. 测试声明
 
 为了执行单元测试你需要先拉取测试仓库
diff --git a/src/page/page.cc b/src/page/page.cc
index 4e2100a..88f258c 100644
--- a/src/page/page.cc
+++ b/src/page/page.cc
@@ -26,11 +26,11 @@ void Page::SetPageID(PageID nPageID) {
   this->_bModified = true;
 }
 
-void Page::GetHeader(uint8_t *dst, PageOffset nSize, PageID nOffset) const {
+void Page::GetHeader(uint8_t *dst, PageOffset nSize, PageOffset nOffset) const {
   MiniOS::GetOS()->ReadPage(_nPageID, dst, nSize, nOffset);
 }
 
-void Page::SetHeader(const uint8_t *src, PageOffset nSize, PageID nOffset) {
+void Page::SetHeader(const uint8_t *src, PageOffset nSize, PageOffset nOffset) {
   MiniOS::GetOS()->WritePage(_nPageID, src, nSize, nOffset);
   this->_bModified = true;
 }
diff --git a/src/page/page.h b/src/page/page.h
index 2b10fa0..4a2e40e 100644
--- a/src/page/page.h
+++ b/src/page/page.h
@@ -19,24 +19,24 @@ class Page {
 
   /**
    * @brief 读出无格式页面头部分的内容。
-   * 
+   *
    * @param dst 读出内容存放地址
    * @param nSize 读出内容长度
-   * @param nOffset 读出内容在头部分起始位置 
+   * @param nOffset 读出内容在头部分起始位置
    */
-  void GetHeader(uint8_t *dst, PageOffset nSize, PageID nOffset) const;
+  void GetHeader(uint8_t *dst, PageOffset nSize, PageOffset nOffset) const;
   /**
    * @brief 写入无格式页面头部分的内容。
-   * 
+   *
    * @param src 写入内容存放地址
    * @param nSize 写入内容长度
    * @param nOffset 读出内容在头部分起始位置
    */
-  void SetHeader(const uint8_t *src, PageOffset nSize, PageID nOffset);
+  void SetHeader(const uint8_t *src, PageOffset nSize, PageOffset nOffset);
 
   /**
    * @brief 读出无格式页面数据部分的内容。
-   * 
+   *
    * @param dst 读出内容存放地址
    * @param nSize 读出内容长度
    * @param nOffset 读出内容在数据部分起始位置
@@ -44,7 +44,7 @@ class Page {
   void GetData(uint8_t *dst, PageOffset nSize, PageOffset nOffset) const;
   /**
    * @brief 写入无格式页面数据部分的内容。
-   * 
+   *
    * @param src 写入内容存放地址
    * @param nSize 写入内容长度
    * @param nOffset 读出内容在数据部分起始位置
